Bootstrap.php

```diff
@@ -796,6 +796,49 @@ private function removeMyAttribute(array $attribute)
         }
     }
 
+    /**
+     * Check if plugin installation path is valid and updates DB config if it is not.
+     *
+     * @return void
+     * @throws ReflectionException|Zend_Db_Adapter_Exception
+     */
+    private function validatePathSource()
+    {
+        // Check that the path is valid
+        $reflection = new \ReflectionClass($this);
+        if ($fileName = $reflection->getFileName()) {
+            $dirName = dirname($fileName) . DIRECTORY_SEPARATOR;
+            if ($this->Path() === $dirName) {
+                return;
+            }
+        }
+        $this->updatePluginSource();
+    }
+
+    /**
+     * @throws Zend_Db_Adapter_Exception
+     */
+    private function updatePluginSource()
```

Comment:
Is this really the norm on Shopware? I'm afraid, we'll push out changes that don't factor in edge cases